### PR TITLE
[Docs] Add like, regexp function documents

### PR DESCRIPTION
**Pull request by EmmyMiao87**
## Proposed changes

[Docs] Add like, regexp function documents

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

### CODE REVIEW

File: docs/.vuepress/sidebar/en.js [hunk 1]
```diff
@@ -323,6 +321,24 @@ module.exports = [
               "starts_with",
               "strleft",
               "strright",
+              {
```
Comment (morningman) [inline docs/.vuepress/sidebar/en.js:324]: Do we need an extra level?

**Review by morningman — COMMENTED** [review]
The CI failed, maybe you need to build the doc in your local env to see what happens.

File: docs/.vuepress/sidebar/en.js (hunk 1 above)
Comment (EmmyMiao87) [reply docs/.vuepress/sidebar/en.js:324]: It’s better to put like and not like together

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**